src/Symfony/Component/Filesystem/Exception/RuntimeArgumentException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Filesystem\Exception;
+
+/**
+ * @author Théo Fidry <theo.fidry@gmail.com>
+ */
+class RuntimeArgumentException extends \RuntimeException implements ExceptionInterface
```

Comment:
This exception does not seem to be used, do I miss something?

Reply:
Hm I think it was meant to replace the usage of `\RuntimeException` in `Path`. I renamed it and replace the usage